oaknational/Oak-Web-Application
# Switch to using `subjectPhaseOptions` so we have `ks4_options`

## Description
Previously we were using `subjectPhaseOptionsIncludeNew` which doesn't include `ks4_options` changes. We can switch to `subjectPhaseOptions` as we no longer require the "new" state

## Issue(s)

Fixes `CUR-983`

## How to test

1. Go to https://deploy-preview-2891--oak-web-application.netlify.thenational.academy/teachers/curriculum/
2. Test downloads still work as expected with correct examboard/pathway in cover/filepath
